bootstrap/handlers/httpserver.go

```diff
@@ -218,15 +218,18 @@ func (b *HttpServer) BootstrapHandler(
 
 			ozServiceName := zerotrust.OpenZitiServicePrefix + b.serverKey
 			lc.Infof("Using OpenZiti service name: %s", ozServiceName)
-			ln, listenErr := zitiCtx.Listen(ozServiceName)
-			if listenErr != nil {
-				err = fmt.Errorf("could not bind service " + ozServiceName + ": " + listenErr.Error())
-				break
+			for t.HasNotElapsed() {
+				ln, listenErr := zitiCtx.Listen(ozServiceName)
+				if listenErr != nil {
+					err = fmt.Errorf("could not bind service " + ozServiceName + ": " + listenErr.Error())
+					t.SleepForInterval()
+				} else {
+					zc.c = &zitiCtx
+					lc.Infof("listening on overlay network. ListenMode '%s' at %s", listenMode, addr)
+					err = server.Serve(ln)
+					break
+				}
```

Comment:
Please handle the situation when the timeout reach but still got the listenErr
At least print an error log